### PR TITLE
Add react-router

### PR DESCRIPTION
**Pull request by earnubs**
Add basic routes with react-router, also on server side.

NB. testing here is not sufficient, but we need jsdom or something to get a DOM and be more rigorous in the tests, I've added a card on Trello.

### CODE REVIEW

File: src/routes.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Route, IndexRoute } from 'react-router';
+import App from './containers/app.js';
+import About from './containers/about.js';
+import Home from './containers/home.js';
+
+export default (
+  <Route path="/" component={App}>
```
Comment (bartaz) [inline src/routes.js]: Writing routes in "HTML" ... that's something to get used to :)

File: server/server.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
 import Express from 'express';
 import React from 'react';
-import ReactDOM from 'react-dom/server';
+import {renderToString} from 'react-dom/server';
+import { match, RouterContext } from 'react-router';
```
Comment (bartaz) [inline server/server.js]: I noticed we are bit inconsistent in using brackets in imports (and other places).
I don't know if eslint can lint that. But maybe something to consider.

**Comment by bartaz** [comment]
Looks good, and it's pre-rendered on server side! 👏

File: server/server.js (hunk 2 above)
Comment (earnubs) [inline server/server.js]: Yep http://eslint.org/docs/rules/object-curly-spacing

My vote is { foo }, but I'll not break down in tears if we vote for {foo} ...

Comment (katiefenn) [inline server/server.js]: +1 { foo }

**Pull request merged**